actions/dump-charm-debug-artifacts/action.yml

```
@@ -12,7 +12,7 @@ runs:
       id: mkdir
       shell: bash
       run: |
-        LOG_DIR="tmp_logs"
+        LOG_DIR="${LOGDUMP_OUTPUT_DIR:-tmp_logs_$GITHUB_JOB}"
```

If not setting `LOGDUMP_OUTPUT_DIR`, will the log dir be `:-tmp_logs_$GITHUB_JOB`?
Also, I think you want to do the following:
```suggestion
        LOG_DIR="$LOGDUMP_OUTPUT_DIR:-tmp_logs_$GITHUB_JOB"
```

---

Yes - basically `tmp_logs_$GITHUB_JOB` is the fallback if `LOGDUMP_OUTPUT_DIR` is not set. The log files will not overwrite each other even without `LOGDUMP_OUTPUT_DIR` since `$GITHUB_JOB` is an unique identifier.


I think we need the brackets. A local test in my bash shell yields:
```
// LOGDUMP_OUTPUT_DIR is not set --> fallback to `tmp_logs_$GITHUB_JOB`
➜ echo $LOGDUMP_OUTPUT_DIR         

➜ GITHUB_JOB=test 

➜ LOG_DIR=${LOGDUMP_OUTPUT_DIR:-tmp_logs_$GITHUB_JOB}        

➜ echo $LOG_DIR
tmp_logs_test

// With LOGDUMP_OUTPUT_DIR set:
➜ LOGDUMP_OUTPUT_DIR=foo                             

➜ LOG_DIR=${LOGDUMP_OUTPUT_DIR:-tmp_logs_$GITHUB_JOB}

➜ echo $LOG_DIR                                      
foo

// Without brackets:
➜ LOG_DIR=$LOGDUMP_OUTPUT_DIR:-tmp_logs_$GITHUB_JOB  

➜ echo $LOG_DIR                                    
foo:-tmp_logs_test
```